.gitignore

```diff
@@ -12,3 +12,5 @@ pre-commit
 
 *.log
 htmlcov/
+
+.vscode/*
```

Comment:
I would keep it, sure makes sense to ignore such files globally on your computer, but you never know who decides to contribute, I feel like it's just easier to keep it there.

I think this happened because I used different computer to commit the changes, hence had a different config.